way-cooler/config/init.lua

```diff
@@ -114,3 +114,9 @@ end
 
 -- !! Do not place any code after this comment.
 -- !! way-cooler and plugins may insert auto-generated code.
+local status = os.execute("which way-cooler-bg 2>/dev/null")
```

Comment:
Did you want to put this under the `!! Do not place any code after this comment` comment? Do you want a `way-cooler.exec(path)` Lua method for the init file? 
This is something in the init file we'll have to change a lot - we may be running the background as an IPC client with IPC privileges in the future, and we won't want to be `print`ing things once we have graphical error messages. Obviously that is very far in the future. 
Do you want to move this to a wrapped Lua method, like `way-cooler.set_background(image_or_color)` which in Rust handles finding and executing `way-cooler-bg` and logging errors? It might help us change the way background is set internally without breaking backwards compatibility in the init file.